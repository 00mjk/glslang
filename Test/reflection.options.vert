#version 440 core

struct VertexInfo {
    float position[3];
    float normal[3];
};

struct TriangleInfo {
    VertexInfo v[3];
};

buffer VertexCollection {
    TriangleInfo t[5];
};

void main()
{
    float f;
    f += t[0].v[0].position[0];
    f += t[gl_InstanceID].v[gl_InstanceID].position[gl_InstanceID];
    f += t[gl_InstanceID].v[gl_InstanceID].normal[gl_InstanceID];
    TriangleInfo tlocal[5] = t;
}
