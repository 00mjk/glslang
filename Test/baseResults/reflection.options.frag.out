reflection.frag
Uniform reflection:

Uniform block reflection:

Pipeline input reflection:
inval: offset 0, type 1406, size 0, index 0, binding -1, stages 0

Pipeline output reflection:

